extension.js

```diff
@@ -0,0 +1,87 @@
+var successColor = '#5cb85c';
+var warningColor = '#f0ad4e';
+var errorColor = '#d9534f';
+var okLimit = 1;
+var warningLimit = 3;
+var pollInterval = 1;   //in minutes
+var issuesURL = 'https://api.github.com/orgs/renuo/issues';
```

Comment:
Could you make this configurable too? Otherwise it's usable by nobody else than us.